encord/transformers/coco/coco_datastructure.py

```diff
@@ -90,12 +91,19 @@ def as_dict_custom(data_class):
             add_rotation = value
         if key == "classifications":
             add_classifications = value
+        if key == "manual_annotation":
+            add_manual_annotation = value
 
     if add_id_value is not None:
         res["id"] = add_id_value
     del res["id_"]
 
-    if add_track_id is not None or add_rotation is not None or add_classifications is not None:
+    if (
```

Comment:
You can probably flip this into creating the attributes dict and then conditionally adding things to it, and have a small check at the end which sets it if it's not empty.
```
attributes = {}
    if add_track_id is not None:
        attributes["track_id"] = add_track_id
    del res["track_id"]
    ....
    
    if not attributes:
      res["attributes"] = attributes
```

Reply:
Nice one, I addressed it @rad-cord  (even though the boolean was flipped)
![disney-auctions-cheshire-cat-pointing_1_9cc11279b5925092a7039c794c2d248e](https://user-images.githubusercontent.com/94055292/204787888-a659e347-f1c2-4a8d-8f1a-ce2369cee163.jpeg)